vmm/src/cpu.rs

```diff
@@ -1024,14 +1047,22 @@ impl CpuManager {
         Ok(())
     }
 
+    pub fn create_boot_vcpus(&mut self, entry_point: EntryPoint) -> Result<()> {
+        self.create_vcpus(self.boot_vcpus(), Some(entry_point))
+    }
+
     // Starts all the vCPUs that the VM is booting with. Blocks until all vCPUs are running.
-    pub fn start_boot_vcpus(&mut self, entry_point: EntryPoint) -> Result<()> {
-        self.activate_vcpus(self.boot_vcpus(), Some(entry_point))
+    pub fn start_boot_vcpus(&mut self) -> Result<()> {
+        self.activate_vcpus(self.boot_vcpus(), false)
     }
 
     pub fn resize(&mut self, desired_vcpus: u8) -> Result<bool> {
         match desired_vcpus.cmp(&self.present_vcpus()) {
-            cmp::Ordering::Greater => self.activate_vcpus(desired_vcpus, None).and(Ok(true)),
+            cmp::Ordering::Greater => {
+                self.create_vcpus(desired_vcpus, None).unwrap();
+                self.activate_vcpus(desired_vcpus, true).unwrap();
```

Comment:
Please handle the `Error` with `map_err()` instead of `unwrap()`.

Reply:
Absolutely, it shouldn't panic here with `unwrap()`.